README.md

```diff
@@ -122,6 +129,11 @@ Update the `composer.json`:
     composer update --lock
   ```
 
+Update the `package.json`:
+
+* Change the `name` from `palantirnet--drupal-skeleton` to `palantirnet--PROJECTNAME`
+* Update the `description` with a brief description of your project.
+
```

Comment:
This is a new addition.

Reply:
@todo: add a link to name rules here: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#name